cumulusci/cumulusci.yml

```diff
@@ -286,18 +286,6 @@ tasks:
         options:
             src_dir: src
         group: Salesforce DX
-    dx_pull:
```

Comment:
@jstvz Why was this done? These are incredibly important and useful commands all over the documentation and is the proper way to interact with scratch orgs.

Reply:
@yippie We removed the `dx_pull` and `dx_push` commands because the underlying Salesforce CLI commands they relied on—`force:source:pull` and `force:source:push`—have been deprecated by Salesforce and replaced with `project retrieve start` and `project deploy start`.

These new commands behave differently, especially in how they handle source tracking and conflict resolution. Keeping the old `dx_pull` and `dx_push` commands could mislead users into thinking everything works the same, which isn’t the case. We want to avoid confusion and ensure everyone updates their workflows to use the new commands.

We recommend updating your scripts to use `project retrieve start` and `project deploy start` directly through the `dx` task or a custom task. We thought it safest to encourage users to verify that these new commands work as expected in their environments.